buildSrc/src/main/kotlin/io/getstream/chat/android/Dependencies.kt

```diff
@@ -124,6 +125,7 @@ object Dependencies {
     const val coroutinesTest = "org.jetbrains.kotlinx:kotlinx-coroutines-test:${Versions.COROUTINES}"
     const val constraintLayout = "androidx.constraintlayout:constraintlayout:${Versions.CONSTRAINT_LAYOUT}"
     const val dexter = "com.karumi:dexter:${Versions.DEXTER}"
+    const val dagCommand = "com.github.leandroborgesferreira:dag-command:${Versions.DAG_COMMAND}"
```

Comment:
Should we deploy it in our nexus before using it?

Reply:
As it is not a plugin was developed inside GetStream, I think it is fine to use without deploy into our nexus